### PR TITLE
Add Github Actions CI

### PR DESCRIPTION
**Pull request by jmleroux**
Remove Circle CI in favor of Github Actions.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
         }
     ],
     "require": {
-        "php": ">=7.2",
+        "php": "^7.2||8.0.*",
```
Comment (jmleroux) [inline composer.json:16]: PHP 8.1 is not compatible for the moment

**Review by ahocquard — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+name: Akeneo Coupling Detector CI
+on: [push]
+jobs:
+    build_and_test:
+        runs-on: ubuntu-latest
+        strategy:
+            matrix:
+                php-versions: ['7.2', '7.3', '7.4', '8.0']
```
Comment (ahocquard) [inline .github/workflows/ci.yml:8]: I would remove 7.2 and 7.3, not maintained anymore

Comment (jmleroux) [reply .github/workflows/ci.yml:8]: Indeed, but I also would like to remove them from the composer.json to "officially" not support them anymore.

Comment (jmleroux) [reply .github/workflows/ci.yml:8]: And yet, we still use the coupling on PIM 4.0, which still uses 7.3 :no_mouth:

**Review by juliensnz — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 2 above)
Comment (ahocquard) [reply .github/workflows/ci.yml:8]: Just tag it :)

**Pull request merged**